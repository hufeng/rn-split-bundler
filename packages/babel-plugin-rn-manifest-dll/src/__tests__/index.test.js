const rnManifestDll = require('../index');
const babel = require('babel-core');

const example = `
  var React = require("react");
  var ReactNative = require("react-native");
`;

it('transform manifest', () => {
  const {code} = babel.transform(example, {
    plugins: [
      [
        rnManifestDll,
        {
          manifeshFilePath: './example/TesterApp/manifest.json',
        },
      ],
    ],
  });
  expect(code).toMatchSnapshot();
});
