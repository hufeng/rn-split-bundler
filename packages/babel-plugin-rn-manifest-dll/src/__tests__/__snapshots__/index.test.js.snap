// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transform manifest 1`] = `
"
var React = qmrequire(500379870);
var ReactNative = qmrequire(1896797308);"
`;
